examples/thermosol/README.md

```diff
@@ -0,0 +1,15 @@
+# Thermodynamic Solubility
+
+Thermodynamic solubility is interesting from a research perspective because it's quite hard to score 
+high on this metric, even though the assay is relatively low-noise. 
+
+The dataset contains data on thermodynamic solubility, pKa-A1 and pKa-B1. The dataset on thermodynamic
+solubility contains 1743 examples, while the others contains 200 examples.
+
+- dataset: {thermosol.csv, pKa-A1.csv, pKa-A2.csv}
+
```

Comment:
do we know units of score?  What assay conditions were etc here?

Reply:
that's a question worth answering. let me pull them.

Reply:
Thermosol: ASTRAZENECA: Solubility in pH7.4 buffer using solid starting material using the method described in J. Assoc. Lab. Autom. 2011, 16, 276-284. Experimental range 0.10 to 1500 uM

pKa-A1: Most acidic pKa value (pKa A1) determined by absorption and potentiometric titration using standard methodology from Sirius Analytical. Experimental range Acids: <= 11.

pKa-B1: ASTRAZENECA: Most basic pKa value (pKa B1) determined by absorption and potentiometric titration using standard methodology from Sirius Analytical. Experimental range Bases: >= 2.